charts/victoria-logs-single/values.yaml

```diff
@@ -364,6 +364,16 @@ fluent-bit:
           Operation    lift
           Nested_under kubernetes
           Add_prefix   kubernetes_
+      [FILTER]
+          Name          parser
+          Match         *
+          Parser        json
+          Key_Name      log
+          Reserve_Data  On
+      [FILTER]
+          Name   modify
+          Match  *
+          Rename log msg
```

Comment:
Will this affect log lines which are not formatted in json?
Will a parser just skip non-json entries?

Reply:
it extracts parsed entries only and removes original fields it it parsed successfully

Reply:
It would be great to clarify this in changelog as the data format will change and users will need to take an action to use an updated data format.